### DISCUSSION TITLE
Fantastic work

### DISCUSSION THREAD

**Discussion by jamiejk** [thygate/stable-diffusion-webui-depthmap-script#165] [General]
Hey, this is absolutely fantastic work. Thanks so much for putting this together. Just got it working and I'm impressed with the results to say the least.

Can I ask what is in the pipeline, features-wise?

**Comment by thygate** [comment]
Thanks. In the short term only custom depthmap input for new features .. Any suggestions ?

**Comment by jamiejk** [comment]
I'll resist suggesting things until I've explored all the parameters
available right now :)

On Sat, 8 Apr 2023 at 06:41, Bob Thiry ***@***.***> wrote:

> Thanks. In the short term only custom depthmap input for new features ..
> Any suggestions ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/thygate/stable-diffusion-webui-depthmap-script/discussions/165#discussioncomment-5559952>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABW2YQG2UB3467FHJAMABXLXAFTG3ANCNFSM6AAAAAAWXEC3ZY>
> .
> You are receiving this because you authored the thread.Message ID:
> <thygate/stable-diffusion-webui-depthmap-script/repo-discussions/165/comments/5559952
> @github.com>
>


-- 
_____________________________________________________________

Jamie King
                   ***@***.***

**Reply by jamiejk** [reply]
I mean, all the stuff you mentioned in the thread "https://github.com/thygate/stable-diffusion-webui-depthmap-script/discussions/50" looks good. If Parallax mapping is low-hanging fruit, it would be great to have!

**Comment by disk02** [comment]
I wanted to chime in and say I agree. This extension is awesome. Thanks for all the effort.
